src/core/frameworks/gql/queries.ts

```diff
@@ -49,7 +49,9 @@ const BASE_POSITION_FRAGMENT = gql`
   }
 `;
 
-// lewv = line event with value
+// TODO: This fragment is broken and needs to be fixed to be used again.
```

Comment:
I'm don't know why, but I cannot get the `LINE_EVENT_FRAGMENT` to function appropriately within queries. Would be great to debug this in a pair programming session with someone!

Reply:
all the values exist on the subgraph schema. are any of the values null?

Reply:
This was fixed by adding `possibleTypes.json` into the the `InMemoryCache` within the `frameworks/gql/index.ts` file: 

![image](https://user-images.githubusercontent.com/11972548/207742235-2d8137ae-cad9-4394-8be9-f50511bc5bda.png)